docs/en/operation/Journal.md

```diff
@@ -75,6 +75,30 @@ alluxio.master.backup.directory=/alluxio/backups
 See the [backup command documentation]({{ '/en/operation/Admin-CLI.html' | relativize_url }}#backup)
 for additional backup options.
 
+### Automatically backing up the journal
+
+Alluxio supports automatically taking primary master metadata snapshots every day at a fixed time 
+so that Alluxio metadata can be restored to at most one day before.
+This functionality is enabled by setting the following property in `conf/alluxio-site.properties`:
+
+```
+alluxio.master.daily.backup.enabled=true
+```
+
+The time to take daily snapshots is defined by `alluxio.master.daily.backup.time`. For example, if 
+a user specified `alluxio.master.daily.backup.time=5:30`, Alluxio primary master will back up its metadata at
+`alluxio.master.backup.directory` of Alluxio root under filesystem every day at `5:30`. 
```

Comment:
.. , the Alluxio primary master will back up its metadata to the `alluxio.master.backup.directory` directory of the root UFS every day at 5:30am UTC.

Reply:
Changed, thanks